pom.xml

```diff
@@ -788,7 +788,7 @@ https://gitbox.apache.org/repos/asf?p=maven-pmd-plugin.git;a=blob_plain;f=src/ma
               <configuration>
                 <deploymentName>SAP AI SDK ${project.version}</deploymentName>
                 <publishingServerId>central</publishingServerId>
-                <autoPublish>false</autoPublish>
+                <autoPublish>true</autoPublish>
```

Comment:
(Question)

What does this change do?

Reply:
You don't have to click the publish button in central sonatype

Reply:
👍